helm/templates/processbot.yaml

```diff
@@ -49,8 +49,8 @@ spec:
         - name: processbot-key-volume
           mountPath: "/etc/processbot-key"
           readOnly: true
-        - name: database
-          mountPath: /usr/local/share/db
+        - name: storage
+          mountPath: {{ .Values.app.STORAGE_PATH }}
```

Comment:
@paritytech/devops Is this a fine approach? I'm proposing this change because it would be better to put `$REPOSITORIES_PATH` (the repositories were processbot will use to clone repositories) in a persistent storage to avoiding cloning repositories from scratch every time the bot restarts. Substrate and Polkadot take a long time to clone (this delay is relevant for the processbot merges, I would like to avoid it if possible) and the repositories end up using lots of space. Currently the git folder alone:

```
> du -h ./polkadot/.git
483M	./polkadot/.git

> du -h ./substrate/.git
234M	./substrate/.git
```

Reply:
IMO it's fine to have it as a variable since the volume is already created and this only makes use of the volume to store the git repos. However, the name can't be changed to `storage`, because it's been already created with the name `database` in `volumes` [section](https://github.com/paritytech/parity-processbot/pull/359/files#diff-00593fb7c733c4e7a93d5ebae69d3a7eff2d122be2b99c90bb6adb6a60a433f1L52) of the pod.

I would also suggest changing the variable name to `storagePath` to match helm chart best practices.

git shallow clone might also help here to prevent downloading the entire git history and only fetch the latest commit, not sure if that helps at all in your case:
```
git clone -–depth [depth] [remote-url]
```
You can also use git shallow clone to access a single branch:
```
git clone [remote-url] --branch [name] --single-branch [folder]
```

Reply:
forget about changing the variable name to `storagePath` as I see other variables are not following this naming convention.

Reply:
> forget about changing the variable name to `storagePath` as I see other variables are not following this naming convention.

It's because the other variables are coming from Gitlab environment variables. I think it's clearer to keep the same name in that case.